apt/private/nested_dict.bzl

```diff
@@ -0,0 +1,45 @@
+"nested dict"
+
+def _set(store, keys, value, add = False):
```

Comment:
We can revert this, this is more involved than i would like the util .set/get_dict to be.

Reply:
I thought it made sense to have it as a first-class data struct rather than just a helper function on a dict. The struct ensures the access to the dict is through the get-set while just having the functions doesn't prevent any direct change to the dict.

Also, I was using the functionality in other places in the code (e.g. the lockfile refactor) so having it all encapsulated in one data struct made more sense to me.

Do you still prefer it back to naked functions over a dict?